spec/defines/keepalived_vrrp_sync_group_spec.rb

```diff
@@ -197,6 +197,24 @@
             )
         }
       end
+
+      describe 'with parameter track_interface' do
+        let(:params) do
+          {
+            group: '_GROUP_',
+            track_interface: ['_VALUE_']
+          }
+        end
+
+        it { is_expected.to create_keepalived__vrrp__sync_group('_NAME_') }
+
+        it {
+          is_expected.to \
+            contain_concat__fragment('keepalived.conf_vrrp_sync_group__NAME_').with(
+              'content' => %r{_VALUE_}
+            )
```

Comment:
Although this will pass, `_VALUE_` is kinda common and doesn't actually test `track_interfaces` is included in the config as expected.

You could use a more restrictive regex to catch both `track_interface` and the actual value

Reply:
Indeed, that could be better.
Fixed with a proper regex in commit [013132a](https://github.com/voxpupuli/puppet-keepalived/pull/264/commits/013132aa9714b32adbefb8c68b426f7bbc6286b0)